NoQuD/homogenize/homogenize.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python
+
+import numpy as np
+from NoQuD.read_input_data import read_csv_input_file as read_csv
+from NoQuD.step_characteristic_solve.StepCharacteristicSolver import *
```

Comment:
I've been told it is dangerous to import as *, maybe consider importing as a `scs` or something to let the reader know what variables/functions are coming from you step characteristic  sovle